creator-node/src/services/sync/syncImmediateQueue.js

```diff
@@ -0,0 +1,77 @@
+const Bull = require('bull')
```

Comment:
new queue so we can have dedicated concurrency limits for recurring/manual immediate syncs